### PR TITLE
VMware UI tests ignored

### PR DESCRIPTION
**Pull request by GoebelL**
**Changes in this Pull Request:**
- Disabled VMware tests (needs fix of #841)

**Mandatory checks:**
- [X] Branch is up to date (latest changes were fetched from the upstream branch before creating this PR)
- [X] Changes were tested manually
- [ ] Java code changes are unit tested
- [ ] Webtests are successful

**To be checked by the reviewer:**
- [ ] Clean Java code and unit tested changes 
- [ ] UI customizablity is preserved: No hard-coded styling and URL references in JSF views 

**To be checked by the pull request owner:**
- [ ] .MASTER/job/BES_MASTER_IT/
- [ ] .MASTER/job/OSCM_WS_Tests_INTERNAL/
- [ ] .MASTER/job/OSCM_WS_Tests_OIDC/
- [ ] .MASTER/job/OSCM_Integration_Tests_INTERNAL/
- [ ] .MASTER/job/OSCM_Integration_Tests_OIDC/

**Cross repository and core changes:**
- [ ] Interdependent changes have been communicated to the team (if applicable)
- [ ] This pull request includes high risk modifications or core changes (e.g API, datamodel, authentication...). Mandatory reviewers: @goebell or @kowalczyka.

**OSCM Build References:**
- N/A

**Screenshot (if applicable):**
- N/A

---
This change is [Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/847)

### CODE REVIEW

File: oscm-ui-tests/src/main/java/org/oscm/webtest/MarketplacePathSegments.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
   public static final String MARKETPLACE_LANDING_PAGE = "playground.jsf";
   public static final String MARKETPLACE_LANDING_PAGE_ID = "playground.jsf?mId=";
   public static final String MARKETPLACE_SERVICES = "playgroundServices.jsf";
+  public static final String MARKETPLACE_ACCOUNT ="account";
```
Comment (Piotr-K8i) [inline oscm-ui-tests/src/main/java/org/oscm/webtest/MarketplacePathSegments.java]: it should be MARKETPLACE_ACCOUNT = "account/playgroundIndex.jsf";

File: oscm-ui-tests/src/test/java/org/oscm/marketplace/MarketplacePlaygroundAccountWT.java [hunk 2]
```diff
@@ -24,6 +25,9 @@
 import org.oscm.webtest.MarketplacePathSegments;
 import org.oscm.webtest.PortalTester;
 
+// FIXME remove ignore and fix
```
Comment (Piotr-K8i) [inline oscm-ui-tests/src/test/java/org/oscm/marketplace/MarketplacePlaygroundAccountWT.java]: After adding the correct constant in MarketplacePathSegments.java, the Ignore tag is not necessary

**Review by Piotr-K8i — CHANGES_REQUESTED** [review]

**Review by GoebelL — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/847)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @KabacinskiPiotr)

___
*[oscm-ui-tests/src/main/java/org/oscm/webtest/MarketplacePathSegments.java, line 16 at r2](https://reviewable.io/reviews/servicecatalog/oscm/847#-M7DHBa9K8Cu1YQGbeOm-r2-16:-M7HPuGe4tWxKh0D9cch:b-896fix) ([raw file](https://github.com/servicecatalog/oscm/blob/8f303ce8efff4104676a2bc45344434a23814012/oscm-ui-tests/src/main/java/org/oscm/webtest/MarketplacePathSegments.java#L16)):*
<details><summary><i>Previously, KabacinskiPiotr wrote…</i></summary><blockquote>

it should be MARKETPLACE_ACCOUNT = "account/playgroundIndex.jsf";
</blockquote></details>

Done.
___
*[oscm-ui-tests/src/test/java/org/oscm/marketplace/MarketplacePlaygroundAccountWT.java, line 28 at r2](https://reviewable.io/reviews/servicecatalog/oscm/847#-M7DQM1jUEYbPX91CDFf-r2-28:-M7HOaxv9ilre2sYpYGu:b-r26l76) ([raw file](https://github.com/servicecatalog/oscm/blob/8f303ce8efff4104676a2bc45344434a23814012/oscm-ui-tests/src/test/java/org/oscm/marketplace/MarketplacePlaygroundAccountWT.java#L28)):*
<details><summary><i>Previously, KabacinskiPiotr wrote…</i></summary><blockquote>

After adding the correct constant in MarketplacePathSegments.java, the Ignore tag is not necessary
</blockquote></details>

Done - without warranty.

**Review by Piotr-K8i — APPROVED** [review]

**Review by GoebelL — COMMENTED** [review]
> Reviewed 1 of 2 files at r1, 2 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/847)* status: all files reviewed, 2 unresolved discussions (waiting on @KabacinskiPiotr)

**Review by GoebelL — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/servicecatalog/oscm/847#-:-M7HQgBl1_pE0iUPWPmu:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @KabacinskiPiotr)

**Pull request merged**